desktop/src/main/java/bisq/desktop/common/utils/GridPaneUtil.java

```diff
@@ -0,0 +1,131 @@
+package bisq.desktop.common.utils;
+
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Pane;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+
+public class GridPaneUtil {
+    public static void setColumnConstraints50percent(GridPane pane) {
+        setGridPaneTwoColumnsConstraints(pane, 50, 50);
+    }
+
+    /**
+     * Set the grid pane with two column constraints
+     */
+    public static void setGridPaneTwoColumnsConstraints(GridPane pane, int percentageCol1, int percentageCol2) {
+        ColumnConstraints col1 = new ColumnConstraints();
+        col1.setPercentWidth(percentageCol1);
+        ColumnConstraints col2 = new ColumnConstraints();
+        col2.setPercentWidth(percentageCol2);
+        pane.getColumnConstraints().addAll(col1, col2);
+    }
+
+    /**
+     * Get a standard 50/50 two column grid pane with the layout specified
+     */
+    public static GridPane getStandardTwoColumnsGridPane(int hGap, int vGap, Insets gridPadding) {
+        return getTwoColumnsGridPane(hGap, vGap, gridPadding, 50, 50);
+    }
+
+    /**
+     * Get a custom two column grid pane with the layout and column widths specified
+     */
+    public static GridPane getTwoColumnsGridPane(int hGap, int vGap, Insets gridPadding, int col1PercentWidth, int col2PercentWidth) {
+        GridPane gridPane = new GridPane();
+        gridPane.setHgap(hGap);
+        gridPane.setVgap(vGap);
+        gridPane.setPadding(gridPadding);
+        if(col1PercentWidth + col2PercentWidth == 100) {
```

Comment:
I'll throw an exception here, did not send another commit to reduce the amount of commits sent already prior to review. I'm trying to build a GridPaneUtil that can standardize creation of any view that need to have two column views (like Learn, Dashboard, Bisq Easy, etc...)